include/math/vector_float.h

```diff
@@ -1115,6 +1115,23 @@ inline float3 xyz(const float4& v) noexcept
 	return float3(v.x, v.y, v.z);
 }
 
+//Converts rodrigues to quat.
+inline quat quat_from_rodrigues(const math::float3& rodrigues) {

```

Comment:
These functions are in the `math` namespace. There is no need to explicitly specify here the namespace name for the math types and functions.